File: UI/src/app/entity/manga/manga-source.ts
import { BaseLookup } from '../base-lookup';

export class MangaSource extends BaseLookup {

	domainUrl: string;
	imageUrl: string;
	
	titleXpath: string;
	imageXpath: string;

	chapterXpath: string;
	chapterTitleXpath: string;
	chapterDateXpath: string;
	chapterHrefXpath: string;

	constructor(entity?) {
		super(entity);
	}
}

File: Maid.Manga.DB/Entity/Manga/MangaSource.cs
using Maid.Core;

namespace Maid.Manga.DB
{
	public class MangaSource : BaseLookup
	{
		public string DomainUrl { get; set; }

		public string TitleXpath { get; set; }

		public string ImageXpath { get; set; }

		public string ChapterXpath { get; set; }

		public string ChapterHrefXpath { get; set; }

		public string ChapterTitleXpath { get; set; }

		public string ChapterDateXpath { get; set; }

		public string ImageUrl { get; set; }
	}
}
